code/modules/power/lighting.dm

```diff
@@ -479,6 +479,18 @@
 	var/b_colour = "#fffee0"
 	var/list/lighting_modes = list()
 	var/sound_on
+	var/random_tone = TRUE
+	var/list/random_tone_options = list(
```

Comment:
Seems like it should be `/static`.

Reply:
Eh, sure. There are no variants to this list right now.